Sources/Logging.php

```diff
@@ -25,15 +25,25 @@
  */
 function truncateArray($arr, $max_length=1900)
 {
-	$curr_length = array_sum(array_map("strlen", $arr));
+	$curr_length = 0;
+	foreach ($arr as $key => $value)
```

Comment:
Just a question RE syntax.... why the lack of { }? Is that the SMF coding style?

Reply:
For one line things, sure.